types/context.go

```diff
@@ -49,8 +49,8 @@ type Context struct {
 	voteInfo             []abci.VoteInfo // Deprecated: use Cometinfo.LastCommit.Votes instead, will be removed after 0.51
 	gasMeter             storetypes.GasMeter
 	blockGasMeter        storetypes.GasMeter
-	checkTx              bool
-	recheckTx            bool // if recheckTx == true, then checkTx must also be true
+	checkTx              bool // Deprecated: use execMode instead, will be removed after 0.51
```

Comment:
As Frodji suggested, we should add the same on the getter as well.

Reply:
done